src/rom3.c

```diff
@@ -1060,7 +1060,7 @@ void sub_800CBE0(u8 a, u8 *b)
     dp01_prepare_buffer(a, gBattleBuffersTransferData, 4);
 }
 
-void EmitChoosePokemon(u8 a, u8 b, u8 c, u8 d, u8 *e)
+void Emitcmd22(u8 a, u8 b, u8 c, u8 d, u8 *e)
```

Comment:
Any reason why you changed this one? Is `EmitChoosePokemon` not correct?

Reply:
I copied all the names from player buffers and applied them to all files. This one wasn't named in the player controller, so I gave it a temporary name. So yeah, you can change it back to EmitChoosePokemon.